packages/module/src/ExpandableText/ExpandableText.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import { Button, Stack, StackItem } from '@patternfly/react-core';
+import sanitizeHtml from 'sanitize-html';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  expandableText: {
+    "& button": {
+      padding: "0"
+    },
+    "& is-inline": { marginRight: [ "10px", "0.625rem" ] }
+  } });
+
+const dangerousHtml = (html: string) => ({ __html: sanitizeHtml(html) });
+
+export interface ExpandableTextCustomButtonProps {
```

Comment:
thinking if this additional interface is needed... why not just att expandToggle and collapseToggle to `ExpandableTextProps` ?

Reply:
Because if they want to pass in a custom button they need to provide a collapse and expand button.  However the custom button is optional in ExpandableTextProps.